dbms/src/Core/Spiller.h

```diff
@@ -127,6 +135,12 @@ class Spiller
     std::atomic<bool> has_spilled_data{false};
     static std::atomic<Int64> tmp_file_index;
     std::vector<std::unique_ptr<SpilledFiles>> spilled_files;
+
+    // Used for the case that spilled blocks containing only constant columns.
+    // Record the rows of these blocks.
+    std::mutex all_constant_mutex;
+    std::vector<UInt64> all_constant_block_rows;
```

Comment:
use `std::vector<std::atomic<UInt64>>` is enough?

Reply:
Compilation errors will be reported because `std::atomic_unint64` is not copyable.